### ISSUE TITLE
path quoting issue

### ISSUE THREAD

**Issue by Enteleform** [lkytal/quickTask#33]
It appears that target paths are not properly quoted before being passed as an argument to their executing program.  I ran into this issue while attempting to execute a Python script, although it may affect other task types as well.

Here is Quick Task's terminal output when attempting to execute a Python script from the `quicktask.showTasks` menu:   
```
C:\Some Path With Spaces> python C:\Some Path With Spaces\__main__.py
python: can't open file 'C:\Some': [Errno 2] No such file or directory
```

**Comment by harshitsilly** [comment]
for me also the same issue. I find out the issue is when you are using git bash as default terminal .. u will be good with command prompt

**Comment by lkytal** [comment]
Shall be fixed in version 3.5.4.
